src/engines/Cesium/Feature/Model/index.tsx

```diff
@@ -50,11 +47,11 @@ export default function Model({
   const coordinates = useMemo(
     () =>
       geometry?.type === "Point"
-        ? geometry.coordinates
-        : property?.location
-          ? [property.location.lng, property.location.lat, property.height ?? 0]
-          : undefined,
-    [geometry?.coordinates, geometry?.type, property?.height, property?.location],
+        ? property?.height === undefined
+          ? geometry.coordinates
+          : [...geometry.coordinates.slice(0, 2), property.height]
+        : undefined,
```

Comment:
```suggestion
       : property?.location
          ? [property.location.lng, property.location.lat, property.height ?? 0]
          : undefined
```
It's actually not need for Visualizer, but maybe better we do the clean up all at once in the future.